karma.conf.js

```diff
@@ -82,7 +83,11 @@ module.exports = function (config) {
             useBrowserName: false
         },
 
-        reporters: ['spec','coverage-istanbul','junit'],
+        mochaReporter: {
```

Comment:
@adamabeshouse what's this? Seems like this might have introduced some double reporting of unit tests see #frontend-devs channel

Reply:
im sure this is to blame... sorry